analyzer/src/funTest/assets/projects/synthetic/poetry-expected-output.yml

```diff
@@ -26,26 +26,23 @@ packages:
 - id: "PyPI::graphviz:0.19.1"
   purl: "pkg:pypi/graphviz@0.19.1"
   authors:
-  - "Sebastian Bank"
+  - "Sebastian Bank <sebastian.bank@uni-leipzig.de>"
   declared_licenses:
   - "MIT"
-  - "MIT License"
   declared_licenses_processed:
     spdx_expression: "MIT"
-    mapped:
-      MIT License: "MIT"
-  description: "Simple Python interface for Graphviz"
+  description: "Graphviz"
   homepage_url: "https://github.com/xflr6/graphviz"
   binary_artifact:
     url: "https://files.pythonhosted.org/packages/9d/fb/886e8ec7862989afc0c35d15813b6c665fe134cc6027cdde2fa300abe9a2/graphviz-0.19.1-py3-none-any.whl"
     hash:
-      value: "c0e5b8e7497c70c0c4cd83cc8c3bdedf"
-      algorithm: "MD5"
+      value: "f34088c08be2ec16279dfa9c3b4ff3d1453c5c67597a33e2819b000e18d4c546"
+      algorithm: "SHA-256"
   source_artifact:
-    url: "https://files.pythonhosted.org/packages/f6/14/aa3ec10e4ab1524ba69f4742ef9cfa01fd668d0840afe5d5e6f708a95031/graphviz-0.19.1.zip"
+    url: ""
```

Comment:
Why's this missing?

Reply:
See the response on another similar comment.